lib/rentlinx/modules/amenityable.rb

```diff
@@ -1,10 +1,16 @@
 module Rentlinx
+  # A module that encapsulates all ammenity related logic for {Rentlinx::Property} and
+  # {Rentlinx::Unit} objects. All of these methods can be called on Properties and Units.
+  #
+  # TODO: Refactor into BaseAble class along with {Linkable} and {Photoable}
   module Amenityable
+    # Posts the object with its associated amenities
     def post_with_amenities
```

Comment:
again, I know you didn't change that, but I think we should get rid of this method